docker/tarfile/src.go

```diff
@@ -184,9 +184,9 @@ func (s *Source) openTarComponent(componentPath string) (io.ReadCloser, error) {
 	return &tarReadCloser{Reader: tarReader, backingFile: f}, nil
 }
 
-// findTarComponent returns a header and a reader matching path within inputFile,
+// findTarComponent returns a header and a reader matching rpath within inputFile,
```

Comment:
I don’t know what “rpath” stands for here (it is clearly not a reference to `realpath(3p)` nor to `ld -rpath`. Why not just say “path”?

Reply:
(It does seem useful to put a warning _somewhere_ to future maintainers to think about the need to use `path.Clean`. But I can’t think of a place that would work.)

Reply:
I want to say `path.Clean(...)`. Do you have a proposal that is more clear? I don't really know the shadowing rules in Go

Reply:
I opted for `componentPath` for the argument/parameter, so that leaves `path` to actually refer to the import.

Reply:
Ah, I didn’t realize that the package name was conflicting.